client/src/notification/notificationReduxFunctions.js

```diff
@@ -1,6 +1,6 @@
 import { createSelector } from 'reselect';
 import { get } from 'lodash';
-import { RESEARCHIFY_API_ERROR } from '../actions/types';
+import { RESEARCHIFY_API_ERROR, SUCCESS_MESSAGE } from '../actions/types';
```

Comment:
Sorry I didn't know where to put this comment. But it seems that the `errorActionCreator` is not used anywhere - can this be deleted or do you see a use for it? IMO, if we don't have usages, it's best to remove it and when we do need it we can add it back in. 

See **YAGNI**: https://martinfowler.com/bliki/Yagni.html

Reply:
Sure, will do